headeronly_src/sqlite3pp.h

```diff
@@ -294,9 +294,12 @@ namespace sqlite3pp
     };
 
     class query_iterator
-      : public std::iterator<std::input_iterator_tag, rows>
     {
      public:
+      using iterator_category = std::input_iterator_tag;
+      using value_type = rows;
+      using difference_type = void;
```

Comment:
using difference_type = std::ptrdiff_t;
      using pointer = rows*;
      using reference = rows&;

Reply:
Remove the #include <iterator> at the top.

Reply:
ok!

Reply:
I dont think it should be value_type = rows* because it causes a build error.
1>Sqllite3ppDriver.cpp
1>d:\Dropbox\src\own\Projects\Sqllite3ppDriver\Sqllite3ppDriver\sqlite3pp.ipp(540): error C2440: 'return': cannot convert from 'sqlite3pp::query::rows' to 'sqlite3pp::query::query_iterator::value_type'
1>d:\Dropbox\src\own\Projects\Sqllite3ppDriver\Sqllite3ppDriver\sqlite3pp.ipp(540): note: No user-defined-conversion operator available that can perform this conversion, or the operator cannot be called
1>d:\Dropbox\src\own\Projects\Sqllite3ppDriver\Sqllite3ppDriver\Sqllite3ppDriver.cpp(32): error C2062: type 'int' unexpected
1>Done building project "Sqllite3ppDriver.vcxproj" -- FAILED.